pho-chat/src/app/layout.tsx

```
@@ -1,11 +1,12 @@
 import type { Metadata } from 'next';
+export { metadata, dynamic, revalidate } from './metadata';
 import { Geist, Geist_Mono } from 'next/font/google';
 import './globals.css';
+import { AppProviders } from './providers';
+
+import dynamic from 'next/dynamic';
+const AppProviders = dynamic(() => import('./providers').then(m => m.AppProviders), { ssr: false });
```

Biome (2.1.2)

[error] Shouldn't redeclare 'AppProviders'. Consider to delete it or rename it.

'AppProviders' is defined here:


(lint/suspicious/noRedeclare)